spec/integration/lhm_spec.rb

```diff
@@ -17,9 +21,11 @@
         t.add_column(:logins, "int(12) default '0'")
       end
 
+      expected_type = mysql_version.start_with?("8.0") ? "int" : "int(12)"
```

Comment:
From [Changes in MySQL 8.0.19](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-19.html):

>Display width specification for integer data types was deprecated in MySQL 8.0.17, and now statements that include data type definitions in their output no longer show the display width for integer types, with these exceptions:
>
>* The type is `TINYINT(1)`. MySQL Connectors make the assumption that `TINYINT(1)` columns originated as `BOOLEAN` columns; this exception enables them to continue to make that assumption.
>
>* The type includes the `ZEROFILL` attribute.
>
>This change applies to tables, views, and stored routines, and affects the output from `SHOW CREATE` and `DESCRIBE` statements, and from `INFORMATION_SCHEMA` tables.
>
>For `DESCRIBE` statements and `INFORMATION_SCHEMA` queries, output is unaffected for objects created in previous MySQL 8.0 versions because information already stored in the data dictionary remains unchanged. This exception does not apply for upgrades from MySQL 5.7 to 8.0, for which all data dictionary information is re-created such that data type definitions do not include display width.